City-of-Helsinki/drupal-hdbt
# UHF-8339 anchor link

# [UHF-8339](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8339)
Anchor links didn't work properly

## What was done
Called scrollIntoView from correct element.

## How to install

* Make sure your instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-8339_anchor_link`
* Run `make drush-cr`

## How to test
- Go to a page with an accordion or create one.
  - Accordion must have text element with multiple heading elements within the text-field
- Select one of the headings and check it's id from html.
- Add an anchor link to the browser's url (helfi/fi/page#anchor_link)
- Close the accordion item
- Reload the page with anchor link

Browser should open the correct accordion item and scroll top of the page to the correct heading.


[UHF-8339]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8339?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
